lib/train/transports/local.rb

```diff
@@ -199,10 +199,12 @@ def start_pipe_server(pipe_name)
               $scriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock($command)
               try {
                 $stdout = & $scriptBlock | Out-String
-                $result = @{ 'stdout' = $stdout ; 'stderr' = ''; 'exitstatus' = 0 }
+                $exit_code = $LastExitCode
+                $result = @{ 'stdout' = $stdout ; 'stderr' = ''; 'exitstatus' = $exit_code }
               } catch {
```

Comment:
I only need the exit code but at some point we should fix #288 fully and get stderr for passing scripts and stdout for failing scripts.
This whole section should maybe change because the `catch` block only applies to PowerShell scripts that `throw`. The catch block isn't entered for PowerShell scripts that "fail" by calling `exit n`

Reply:
For the change that's given here, this seems fine - you're not making the lack of STDOUT/STDERR worse.

Reply:
This looks reasonable as long as you are looking for the last exit code from the most recent Win32 process, and not the last PowerShell command was successful - [use `$?` for that](https://techibee.com/powershell/what-is-lastexitcode-and-in-powershell/1847)

Reply:
I was taking the previous code's use of `0` to assume that it meant exit code and not exit status (in spite of the variable name)